File: app/src/main/java/com/lambdaschool/sprint4challenge_mymovies/activities/MainActivity.java
package com.lambdaschool.sprint4challenge_mymovies.activities;

import android.graphics.Bitmap;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;

import com.lambdaschool.sprint4challenge_mymovies.R;
import com.lambdaschool.sprint4challenge_mymovies.apiaccess.MovieDbDao;
import com.lambdaschool.sprint4challenge_mymovies.apiaccess.MovieOverview;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    ArrayList<MovieOverview> overviews;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        new Thread(new Runnable() {
            @Override
            public void run() {
                overviews = MovieDbDao.searchMovies("batman");
            }
        }).start();
    }
}

File: app/src/main/java/com/lambdaschool/datapersistencesprintchallenge/activities/MainActivity.kt
package com.lambdaschool.datapersistencesprintchallenge.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import com.lambdaschool.datapersistencesprintchallenge.ListRespritory
import com.lambdaschool.datapersistencesprintchallenge.R
import com.lambdaschool.datapersistencesprintchallenge.recycler.MovieAdapter
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {



    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        //TODO 1 created initial view
        ListRespritory.createView()

        // TODO 2 added an adapter to main activity
        recycle.apply {
            setHasFixedSize(true)
            layoutManager = LinearLayoutManager(applicationContext)
            adapter = MovieAdapter(ListRespritory.createList)
        }
    }


}
